administration/src/routes/users/components/RoleSelector.tsx

```diff
@@ -6,36 +6,41 @@ import { useTranslation } from 'react-i18next'
 import FormAlert from '../../../components/FormAlert'
 import { Role } from '../../../generated/graphql'
 import { ProjectConfigContext } from '../../../project-configs/ProjectConfigContext'
+import { useWhoAmI } from '../../../provider/WhoAmIProvider'
 import roleToText from '../utils/roleToText'
 import RoleHelpButton from './RoleHelpButton'
 
 const RoleSelector = ({
-  role,
+  selectedRole,
   onChange,
-  hideProjectAdmin,
 }: {
-  role: Role | null
+  selectedRole: Role | null
   onChange: (role: Role | null) => void
-  hideProjectAdmin: boolean
 }): ReactElement => {
   const { t } = useTranslation('users')
+  const { role: activeRole } = useWhoAmI().me
   const [touched, setTouched] = useState(false)
   const config = useContext(ProjectConfigContext)
-  const showExternalVerifiedApiUser = config.applicationFeature?.applicationUsableWithApiToken && !hideProjectAdmin
-  const showError = role === null && touched
+  const isProjectAdmin = activeRole === Role.ProjectAdmin
+  const showExternalVerifiedApiUser = config.applicationFeature?.applicationUsableWithApiToken && isProjectAdmin
+  const showProjectStoreManager = config.storesManagement.enabled && isProjectAdmin
+  const showError = selectedRole === null && touched
   return (
     <>
       <Stack direction='row'>
         <FormControl fullWidth size='small' required>
-          <InputLabel shrink={role !== null}>{t('selectRole')}</InputLabel>
+          <InputLabel shrink={selectedRole !== null}>{t('selectRole')}</InputLabel>
           <Select
```

Comment:
🔧 You could add an accessibility label with `labelId={labelId}`. The label ID can be generated with [`useId`](https://react.dev/reference/react/useId).